awx/ui_next/src/components/AppContainer/AppContainer.jsx

```diff
@@ -225,17 +129,14 @@ function AppContainer({ i18n, navRouteConfig = [], children }) {
             ouiaId="session-expiration-logout-button"
             key="logout"
             variant="secondary"
-            onClick={handleLogout}
+            onClick={logout}
           >
             {i18n._(t`Logout`)}
           </Button>,
         ]}
       >
-        {i18n._(
-          t`You will be logged out in ${Number(
-            Math.max(Math.floor(timeRemaining / 1000), 0)
-          )} seconds due to inactivity.`
-        )}
+        <Trans>You will be logged out in</Trans> {sessionCountdown}{' '}
+        <Trans>seconds due to inactivity.</Trans>
```

Comment:
Based on another open PR `t` macro will be preferred syntax to select translation. Right @AlexSCorey ?

Reply:
I think we're OK with `<Trans>` (and in fact I prefer it in some cases) -- though the full sentence including the `sessionCountdown` should appear inside one `<Trans>` so it can all be translated together

Reply:
This also needs `plural` because when we get to 1 left we want `You will be logged out in 1 second due to inactivity`

Reply:
👍 Updated timeout message to use Plural macro.